src/Tribe/Main.php

```diff
@@ -802,6 +802,17 @@ public function activation_page() {
 				'welcome_page_title'    => esc_html__( 'Welcome to Event Tickets!', 'event-tickets' ),
 				'welcome_page_template' => $this->plugin_path . 'src/admin-views/admin-welcome-message.php',
 			] );
+
+			tribe_asset(
+				$this,
+				'tribe-tickets-welcome-message',
+				'admin/welcome-message.js',
+				[ 'jquery' ],
+				'admin_enqueue_scripts',
+				[
+					'conditionals' => [ $this->activation_page, 'is_welcome_page' ]
```

Comment:
* :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 62633389365f640edcffb1ac27bb5bcc</sub>

Reply:
```suggestion
					'conditionals' => [ $this->activation_page, 'is_welcome_page' ],
```